images/fcos-bastion-image/root/usr/bin/gen_registry_file.sh

```diff
@@ -32,11 +32,14 @@ http:
     key: /certs/domain.key
 storage:
   delete:
-    enabled: true
+    enabled: false
```

Comment:
Can we keep this true? With maintenance - readonly set to true it will override this setting. The allowing normal deletions may also keep the disk usage low.

Reply:
Thanks @sgoveas , updated. Please have a another review.